Sources/Utils/Constants.swift

```diff
@@ -58,6 +60,23 @@ struct Constants {
         static let variableType = "variableType"
         static let variableValue = "variableValue"
         static let variableValues = "variableValues"
+        
```

Comment:
am assuming, `variableType` and all above are not valid for decide api case.?

Reply:
Correct. not used for decide-api.